I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 34, in script
    $ resultado = selectFromBD("trilhadobrasil")
  File "game/script.rpy", line 34, in <module>
    $ resultado = selectFromBD("trilhadobrasil")
  File "game/bd.rpy", line 50, in selectFromBD
    cursor.execute(sql,val)
ProgrammingError: Could not process parameters: int(3), it must be of type list, tuple or dict

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 34, in script
    $ resultado = selectFromBD("trilhadobrasil")
  File "C:\Users\Etec Paulino Botelho\Downloads\renpy-8.3.0-sdk\renpy\ast.py", line 821, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Etec Paulino Botelho\Downloads\renpy-8.3.0-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 34, in <module>
    $ resultado = selectFromBD("trilhadobrasil")
  File "game/bd.rpy", line 50, in selectFromBD
    cursor.execute(sql,val)
  File "python-packages/mysql/connector/cursor.py", line 526, in execute
ProgrammingError: Could not process parameters: int(3), it must be of type list, tuple or dict

Windows-10-10.0.19041 AMD64
Ren'Py 8.3.0.24082114
Trilhas do Brasil 1.0
Fri Aug 30 20:41:27 2024
